spec/api-context-bridge-spec.ts

```diff
@@ -1290,6 +1290,115 @@ describe('contextBridge', () => {
           });
         });
       });
+
+      describe('executeInMainWorld', () => {
```

Comment:
Can you please update `generateTests` so the same "it doesn't leak X" tests run in a service worker environment too. They current run for sandbox + unsandbox. They should run for sandbox + unsandbox + service workers.

Reply:
This is a considerable amount of work to refactor these tests. Do you think this should should block the PR or can it be done in a follow up?